plaidml2/op/op.h

```diff
@@ -172,6 +172,38 @@ inline edsl::Tensor prod(const edsl::Tensor& I, const edsl::Value& axes = edsl::
   return details::op("prod", args).as_tensor();
 }
 
+class relu {
+ protected:
+  edsl::Tensor I_;
+  edsl::Tensor alpha_;
+  edsl::Tensor max_value_;
+  float threshold_ = 0.0;
+
+ public:
+  explicit relu(const edsl::Tensor& I) { I_ = I; }
+
+  relu alpha(const edsl::Tensor& alpha) {
+    alpha_ = alpha;
+    return *this;
+  }
+
+  relu max_value(const edsl::Tensor& max_value) {
+    max_value_ = max_value;
+    return *this;
+  }
+
+  relu threshold(float threshold) {
+    threshold_ = threshold;
+    return *this;
+  }
+
+  edsl::Tensor operator()() {
```

Comment:
Actually I was thinking of a cast operator, not a call operator :)

Reply:
Fixed this